### PR TITLE
First release

### PR DESCRIPTION
**Pull request by Ducatel**
ref #1

### CODE REVIEW

File: doxy-my-docs .py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```
Comment (glc-mvion) [inline doxy-my-docs .py]: Please remove trailing space at the end of file name

**Review by glc-mvion — CHANGES_REQUESTED** [review]

File: doxy-my-docs .py (hunk 1 above)
Comment (Ducatel) [reply doxy-my-docs .py]: Fixed

File: doxy-my-docs.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    doxyfile = AppConfiguration().get_doxygen_config()[configurationEnum.Doxygen.DOXYFILE]
+    logging.debug("Doxyfile: {}".format(doxyfile))
+
+    project_config = AppConfiguration().get_project_config()
+
+    doxygen_config_parser = doxygen.ConfigParser()
+    doxygen_configuration = doxygen_config_parser.load_configuration(doxyfile)
+
+    if configurationEnum.Project.VERSION in project_config:
+        logging.debug("Update PROJECT_NUMBER from {} to {}".format(doxygen_configuration['PROJECT_NUMBER'], project_config[configurationEnum.Project.VERSION]))
```
Comment (Servuc) [inline doxy-my-docs.py:33]: Why use `configurationEnum.Project.VERSION` and not `configurationEnum.Project.NUMBER` ?

Comment (Servuc) [reply doxy-my-docs.py:33]: Host My Docs setting name

**Review by Servuc — APPROVED** [review]
Seem good :smiley_cat:

File: doxy-my-docs.py (hunk 2 above)
Comment (Ducatel) [reply doxy-my-docs.py:33]: Normalized with HostMyDocs config and  for me `Project.NUMBER` is lesser clear than `Project.
VERSION`

**Pull request merged**